init_localization_service_server:
  ros__parameters:
    pose_map_to_station_charge_x: 
    pose_map_to_station_charge_y: 
    pose_map_to_station_charge_yaw: 
